addons/libmaszyna/e3d/e3d_model_manager.gd

```diff
@@ -0,0 +1,45 @@
+@tool
+extends Node
+
+
+
+func _set_owner_recursive(node, new_owner):
+    if not node == new_owner:
+        node.owner = new_owner
+    if node.get_child_count():
+        for kid in node.get_children():
+            _set_owner_recursive(kid, new_owner)
+
+
+func load_model(data_path:String, filename: String) -> E3DModel:
+    var output:E3DModel
+    var path = UserSettings.get_maszyna_game_dir() + "/" + data_path + "/" + filename + ".e3d"
+
+    # check users cache
+    var cached_path = "user://e3d_res_cache/"+data_path+"__"+filename+".res"
```

Comment:
We might probably want to store cache for more things so I'd suggest going with
`"user://cache/e3d/res"` or something similar ;)

Reply:
Done